src/hazelcore/extractions/ocaml/OCamlExtraction_Pat.re

```diff
@@ -0,0 +1,114 @@
+// mark with a constructor to tell apart with typ, exp
+type t =
+  | OCamlPat(string);
```

Comment:
Just use a string.  E.g. `type t = string`.

Reply:
Ur, I remembered that Cyrus says adding a constructor is like a "label" to indicate it's a result of pattern extraction. Could you check the idea with Cyrus?

Reply:
@cyrus-, do you want to chime in on this?

Reply:
Nvm @cyrus-, we will use the constructor.